packages/@ember/-internals/glimmer/lib/component-managers/mount.ts

```diff
@@ -142,20 +146,8 @@ class MountManager extends AbstractManager<EngineState, EngineDefinitionState>
     return tag;
   }
 
-  getDestructor(bucket: EngineState): Option<Destroyable> {
-    let { engine, environment, controller } = bucket;
-
-    if (ENV._DEBUG_RENDER_TREE) {
-      return {
-        destroy() {
-          environment.extra.debugRenderTree.willDestroy(controller);
-          environment.extra.debugRenderTree.willDestroy(bucket);
-          engine.destroy();
-        },
-      };
-    } else {
-      return engine;
-    }
+  getDestroyable(bucket: EngineState): Option<Destroyable> {
+    return bucket.engine;
```

Comment:
I assume the controller is a child of the engine and is registered to it

Reply:
Yes, based on the code prior to this, I believe that must have been true. This is effectively the equivalent to the code it's replacing.